code/monitor/checkmonitor.q

```diff
@@ -249,6 +268,15 @@ statusbyfam:{[f]
   `status`timerstatus xasc select from checkstatus where family in f
  }
 
+
+//Delete rows from Checkstatus 
+deleterows:{[checkids]
+ if[all null checkids; :delete from `checkstatus];
+ delete from `checkstatus where checkid in checkids
+ }
```

Comment:
What's this one for? I don't think we should be deleting out of checkstatus - it will likely screw things up

Reply:
This might be mis-communication - was this the thing I asked for to periodically remove rows over a certain age from the checktracker table?

Reply:
ah i have in my notes to delete rows from checkstatus. i can change it to be that ^